testplan/web_ui/testing/src/Report/BatchReportBeta/state/reportActions/fetchReport.js

```diff
@@ -0,0 +1,133 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import Axios from 'axios';
+import { headers as defaultHeaders } from 'axios/lib/defaults';
+import _ from 'lodash';
+import {
+  getReportUid,
+  getReportIsFetching,
+  getReportDocument,
+  getReportLastFetchError,
+  getReportIsFetchCancelled,
+} from '../reportSelectors';
+import { PropagateIndices } from '../../../reportUtils';
+import { toPlainObjectIn, flattened } from '../../../../Common/utils';
+
+const __DEV__ = process.env.NODE_ENV !== 'production';
+
+let TEST_REPORTS = {};  // eslint-disable-line no-unused-vars
+if(__DEV__) {
+  TEST_REPORTS = {
+    fakeReport: require('../../../../Common/fakeReport'),
+    sampleReports: require('../../../../Common/sampleReports'),
+  };
+}
+
+const axiosDefaultConfig = (() => {
+  const baseURL = 'http://testplan-db.ms.com/api/v1/reports';
```

Comment:
`${window.location.origin}/api/v1/reports`

Reply:
works in prod for Morgan STanley, not in development and perhaps our OSS userbase runs their webserver and API endpoints on different hosts.